ProjectTraning/Task_20_07/FridgeRefreg.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning.Task_20_07
+{
+    public class FridgeRefreg
+    {
+        public event FridgeDelegate Fridge;
+
+        public Enum Status { get; set; } = ModesofOperation.On;
```

Comment:
Incorrect type of properties.
Property should have the same type as stored enum.